lib/default-options.js

```diff
@@ -16,7 +16,8 @@ module.exports = function defaultOptions () {
             'assert.notDeepEqual(actual, expected, [message])',
             'assert.deepStrictEqual(actual, expected, [message])',
             'assert.notDeepStrictEqual(actual, expected, [message])'
-        ]
+        ],
+        additionalMethods: []
```

Comment:
I think `wrapOnlyPatterns` is better name here.
We'll rename this if we find better name.